packages/shared-components/src/api/http.test.ts

```diff
@@ -0,0 +1,95 @@
+import { get, HttpError, MimeType } from "./http";
+import nock from "nock";
+
+describe("http requests", () => {
+  describe("get", () => {
+    beforeEach(() => {
```

Comment:
Gjør dette til en funksjon som kalles der den trengs, i stedet for å kjøre før hver test